src/ReleasePage/releaseEntry.jsx

```diff
@@ -306,6 +306,18 @@ function ReleaseEntry({ profile, currentView }) {
                   <div className="card mb-3">
                     <div className="card-body">
                       <p className="release-description">{release.description}</p>
+                      <p className="release-description file-extraction-instruction">
+                        All bundled datasets are compressed (*.tar.gz) and need to be extracted upon downloading. For MAC
```

Comment:
"MAC" should not be all caps, it should be _Mac_ or _Apple Mac_ or _Apple Mac OS_

Reply:
Thanks for catching that! Fixed.